h/events.py

```diff
@@ -20,6 +20,7 @@ class AnnotationTransformEvent(object):
     annotation just before it is indexed or in other use-cases.
     """
 
-    def __init__(self, request, annotation_dict):
+    def __init__(self, request, annotation, annotation_dict):
         self.request = request
+        self.annotation = annotation
         self.annotation_dict = annotation_dict
```

Comment:
Yup, I seem to remember mentioning it in one of the recent commit messages that we should use some sort of "annotation search" formatters, but that we should do it later. I will make sure to add a eng backlog card for it.